src/testConverter.ts

```diff
@@ -292,7 +292,7 @@ export class TestConverter implements vscode.Disposable {
         break;
     }
 
-    if (evt.message && ((evt.state !== 'errored' && evt.state !== 'failed') || !vscodeTest.uri)) {
```

Comment:
For failed failed/errored tests, the message should already get appended on L291 above. Are they not showing in i your case?

Reply:
The message doesn't seem to appear anywhere in the UI unless a `location` is set on it (which is done for `evt.decorations` above on L285).

Reply:
Hi @connor4312, kindly following up here. I really don't see these messages (without `location` set) anywhere, unless I'm missing something in the UI?

Reply:
Apologies for the delay.

Just giving this a shot on a set of Mocha tests and commenting out L258 `message.location =` above, this is what I see, where messages are appended as decorations to the test. Additionally, the peek opens at the location of the test:

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/2230985/209738106-dee18ecf-9987-4d3b-be87-c855c7bf53c8.png">

However, if the _test_ has no URI, then it is true we don't display anything in VS Code right now.

Reply:
Thanks for following on this @connor4312! Hopefully, you can now see why I wanted to append non-decoration messages to the test output because it would otherwise be lost to the void. This problem plagues me with the CMake Test Explorer extension. Another option would be to add something like this to line 277 above:

```js
if (vscodeTest.uri) {
  message.location = new vscode.Location(vscodeTest.uri, new vscode.Position(0, 0));
}
```

I don't this is approach makes sense though to put faux decorations up at the top of the file instead of appending to the test output.

Reply:
Hi @connor4312, kindly following up on this. Thanks!